compiler/rustc_error_codes/src/error_codes/E0744.md

```diff
@@ -12,12 +12,15 @@ const _: i32 = {
 };
 ```
 
-At the moment, `if` and `match`, as well as the looping constructs `for`,
-`while`, and `loop`, are forbidden inside a `const`, `static`, or `const fn`.
+At the moment, `for` loops, `.await`, and the `Try` operator (`?`) are forbidden
+inside a `const`, `static`, or `const fn`.
```

Comment:
Btw, arguably all of these are indeed control-flow operations, if you dig down into how they are implemented. But I am okay with your rewording, since that is somewhat non-obvious (and since other control-flow operations are supported nowadays).